apps/generic-issuance-client/.env.example

```diff
@@ -1,10 +1,17 @@
 # This token is used to authenticate Stytch requests for user authentication.
-# If you are on the team, message @rrrliu to get one.
-STYTCH_PUBLIC_TOKEN=
+# You can bypass email login in local development by omitting this environment
+# variable, and by setting the `STYTCH_BYPASS=true` environment variable in
+# `passport-server`.
+#
+# STYTCH_PUBLIC_TOKEN=
```

Comment:
if this var is not set, podbox enters a dev-only 'bypass email registration' mode, which works with a correspondingly-configured podbox server on the other end